tests/systems/systems_test_data.py

```diff
@@ -168,3 +168,74 @@ def network_factory(*args: Any, **kwargs: Any) -> Any:
         trainer_parameter_update_period=1,
     )
     return test_system
+
+
+def ippo_system_multi_thread_eval() -> System:
+    """Multi thread IPPO test system using Launchpad"""
+    # Environment.
+    environment_factory = functools.partial(
+        debugging_utils.make_environment,
+        env_name="simple_spread",
+        action_space="discrete",
+    )
+
+    # Networks.
+    def network_factory(*args: Any, **kwargs: Any) -> Any:
+        return ippo.make_default_networks(  # type: ignore
+            policy_layer_sizes=(256, 256, 256),
```

Comment:
Should we make these smaller to aligned with the paper and other examples?

Reply:
Yeah sure!
Also one question from my side should we remove the pettingzoo sequential wrapper? https://github.com/instadeepai/Mava/blob/c23db5e8126d17288a83c3df04d57a41a3b6fb90/mava/wrappers/pettingzoo.py

Reply:
Yes, I vote we remove it. If we aren't planning on using it in the near future, it means we have less code to test/maintain. We can add it back in when needed.